documentation/cli/change_log.rst

```diff
@@ -7,11 +7,15 @@ FlexMeasures CLI Changelog
 since v0.16.0 | September XX, 2023
 =======================================
 * Add command ``flexmeasures add sources`` to add the base `DataSources` for the `DataGenerators`.
-
+* Add ``--kind reporter`` option to create the asset and sensors for the reporter tutorial.
 
 since v0.15.0 | August 9, 2023
 =================================
+* Add ``--id`` option to ``flexmeasures show data-sources`` to show just one ``DataSource``.
+* Add ``--show-attributes`` flag to ``flexmeasures show data-sources`` to select whether to show the attributes field or not.
 
+since v0.15.0 | August 9, 2023
+================================
```

Comment:
The 0.15 header is duplicated. I believe all of the new entries should fall under 0.16.